### PR TITLE
API-16 fix for SkuOption error on Sku resource

### PR DESCRIPTION
**Pull request by zvuki**
## What
Fix for https://github.com/bigcommerce/bigcommerce-api-php/issues/92

## Testing
`Bigcommerce\Api\Client::getSkus()`
more tests here:
https://github.com/bigcommerce/bigcommerce/pull/12071

@aleachjr How do I change a version?

cc @bc-joshroe @lord2800 @becomevocal @ICarpenter @bc-AlyssNoland

### CODE REVIEW

File: test/Unit/Api/Resources/SkuTest.php [hunk 1]
```diff
@@ -25,24 +25,4 @@ public function testUpdatePassesThroughToConnection()
 
         $sku->update();
     }
-
-    public function testOptionsPassesThroughToConnection()
```
Comment (aleachjr) [inline test/Unit/Api/Resources/SkuTest.php]: can we fix/replace this with something else instead of removing?

Comment (zvuki) [inline test/Unit/Api/Resources/SkuTest.php]: options are not passed through to connection anymore, so this expectation does not exist

Comment (aleachjr) [inline test/Unit/Api/Resources/SkuTest.php]: yes but you still have functionality so should have a test of some kind don't you think?

**Comment by bc-sandeep** [comment]
:green_heart: looks grreat

**Comment by aleachjr** [comment]
:+1:

**Pull request merged**